src/inmanta/data/sqlalchemy.py

```diff
@@ -67,12 +68,13 @@ class InmantaModule(Base):
 
     @classmethod
     async def register_modules(
-        cls, environment: uuid.UUID, module_version_info: dict[str, InmantaModuleDTO], connection: asyncpg.Connection
+        cls, environment: uuid.UUID, modules: dict[InmantaModuleName, InmantaModuleDTO], connection: asyncpg.Connection
     ) -> None:
         """
-        Register modules
+        Register Inmanta modules.
```

Comment:
This needs more. It leaves open questions like "what if the modules are already registered"? What if it is already registered for the same version? ...